### PR TITLE
support ppc64le

### LINKED ISSUE #890: Unable to build katib on ppc64le
/kind bug

**What steps did you take and what happened:**
- Dockerfile used to build Katib : 
https://github.com/kubeflow/katib/blob/master/cmd/manager/v1alpha3/Dockerfile

- Got the following error : 
```
Executing busybox-1.30.1-r2.trigger
OK: 118 MiB in 31 packages
can't load package: package github.com/kubeflow/katib/cmd/manager/v1alpha3: cannot find package "github.com/kubeflow/katib/cmd/manager/v1alpha3" in any of:
        /usr/local/go/src/github.com/kubeflow/katib/cmd/manager/v1alpha3 (from $GOROOT)
        /go/src/github.com/kubeflow/katib/cmd/manager/v1alpha3 (from $GOPATH)
```

**Environment:**
- Dockerfile is built on : 
```
# cat /etc/os-release
NAME="Red Hat Enterprise Linux Server"
VERSION="7.6 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.6"
PRETTY_NAME="Red Hat Enterprise Linux Server 7.6 (Maipo)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.6:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 7"
REDHAT_BUGZILLA_PRODUCT_VERSION=7.6
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="7.6"
```

VM : 
9.47.73.84 root / passw0rd
Dockerfile is at /root/

### PR DESCRIPTION
**Pull request by hmtai**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #890 

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/893)

### CODE REVIEW

**Comment by hmtai** [comment]
cc/ @hougangliu Please review.

File: cmd/suggestion/hyperopt/v1alpha3/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,11 @@ RUN if [ "$(uname -m)" = "ppc64le" ]; then \
         pip install cython; \
     fi
 RUN GRPC_HEALTH_PROBE_VERSION=v0.3.0 && \
-    wget -qO/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 && \
+    if [ "$(uname -m)" = "ppc64le" ]; then \
```
Comment (hougangliu) [inline cmd/suggestion/hyperopt/v1alpha3/Dockerfile]: please handle all grpc_health_probe version under `cmd/suggestion` v1alpha3

Comment (hmtai) [reply cmd/suggestion/hyperopt/v1alpha3/Dockerfile]: Done

**Comment by hougangliu** [comment]
@hmtai thanks
Please also update https://github.com/kubeflow/katib/blob/master/cmd/manager/v1alpha3/Dockerfile to use same version grpc_health_probe (follow suggestion way)

**Comment by gaocegege** [comment]
@hmtai @hougangliu 

This PR contains changes in suggestion image build. While now our test infra has some problems on pip install and cannot build the suggestion image successfully. Would you mind testing and merging it after 0.7?

**Comment by hougangliu** [comment]
@gaocegege sure.

**Comment by hougangliu** [comment]
@hmtai please rebase after #907 merged

**Comment by gaocegege** [comment]
Please rebase the master and let's see the CI result

**Comment by hmtai** [comment]
> Please rebase the master and let's see the CI result

Yes,rebase it just now.

**Pull request merged**